app/src/main/res/layout/fragment_add_edit_submission.xml

```diff
@@ -63,22 +63,62 @@
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toBottomOf="@id/new_submission_text" />
 
-        <com.google.android.material.textfield.TextInputLayout
-            android:id="@+id/contact_count_text_input_layout"
-            android:layout_width="80dp"
+        <LinearLayout
+            android:id="@+id/contact_count_layout"
+            android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_marginTop="15dp"
-            android:hint="Contact Count"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toBottomOf="@+id/new_submission_contact_label">
 
-            <com.google.android.material.textfield.TextInputEditText
-                android:layout_width="match_parent"
-                android:layout_height="match_parent"
-                android:inputType="number"
-                android:text="@={viewModel.contactCount}" />
+            <com.google.android.material.textfield.TextInputLayout
+                android:id="@+id/contact_count_text_input_layout"
+                android:layout_width="80dp"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="15dp"
+                android:hint="Contact Count">
 
-        </com.google.android.material.textfield.TextInputLayout>
+                <com.google.android.material.textfield.TextInputEditText
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:inputType="number"
+                    android:text="@={viewModel.contactCount}" />
+
+            </com.google.android.material.textfield.TextInputLayout>
+
+            <androidx.constraintlayout.widget.ConstraintLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent">
+
+                <com.google.android.material.button.MaterialButton
+                    android:id="@+id/decrement_contact_count_button"
```

Comment:
This looks good. We *could* remove the fragment from the click listening entirely and instead use databinding where the code to handle increment/decrement is in the viewmodel. Here, you'd add something like:
`android:onClick="@{() -> viewModel.decrementContactCount()}"

basically the same code that you have but no need to call `setOnClickListener` in the fragment and the logic gets moved to the VM.